docker/minio/init-script.sh

```
@@ -0,0 +1,60 @@
+MINIO_HOST=${MINIO_HOST:-"http://localhost:9000"}
+MINIO_ACCESS_KEY=${MINIO_ACCESS_KEY:-"minioadmin"}
+MINIO_SECRET_KEY=${MINIO_SECRET_KEY:-"minioadmin"}
+
+# Max retries and delay
+MAX_RETRIES=10
+RETRY_COUNT=0
+RETRY_DELAY=5  # 5 seconds delay between retries
+
+# Function to retry a command
+retry_command() {
+    local cmd=$1
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)